src/screens/bolt/index.tsx

```diff
@@ -71,9 +74,11 @@ export default function Bolt() {
     setIsLoading(false)
   }
   useEffect(() => {
+    if (!blsLoaded || isWasmInitializing) return
```

Comment:
What do you think of if bls loaded and wasm is not initializing. call get balance, etc.?